denv.go

```diff
@@ -48,16 +54,52 @@ func main() {
 
 	switch os.Args[1] {
 	case "up":
-		args := extractArgsFromDenvFile()
+		if len(os.Args) < 3 {
+			fmt.Println("Expected service name.")
+			os.Exit(1)
+		}
+
+		denvFile := loadDenvFile("")
+		service, definitionError := getDefinition(os.Args[2], denvFile)
+		if definitionError != nil {
+			os.Exit(1)
+		}
```

Comment:
Since this code is basically the same for te "up" and "down" cases should we put it back in the `extractArgsFromDenvFile` function and rather change the name of it?

Reply:
Yes I though so too, but for the implementation of the `command` feature during the up process, I also need the definition-object.